src/main/java/org/gridsuite/shortcircuit/server/repositories/specifications/AbstractCommonSpecificationBuilder.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.shortcircuit.server.repositories.specifications;
+
+import jakarta.persistence.criteria.Path;
+import jakarta.persistence.criteria.Root;
+import org.gridsuite.shortcircuit.server.dto.ResourceFilter;
+import org.springframework.data.jpa.domain.Specification;
+
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * @author Kevin LE SAULNIER <kevin.lesaulnier@rte-france.com>
+ */
+public abstract class AbstractCommonSpecificationBuilder<T> {
```

Comment:
Unlike in SA, you don't need this class. You extend it only once.

Reply:
Ok I changed my mind and removed it. The code is diverving anyway.